### PR TITLE
TNT map22 and Plutonia map01 screenies

### PR DESCRIPTION
**Pull request by norbcodes**
Hey, me again!

### CODE REVIEW

File: .github/doom_ss/TNT/MAP22/tnt22_4.png
Comment (MF366-Coding) [inline .github/doom_ss/TNT/MAP22/tnt22_4.png:1]: Might be a bit confusing for people to guess but... we have to make this game hard, right?

Comment (MF366-Coding) [reply .github/doom_ss/TNT/MAP22/tnt22_4.png:1]: Also I'm saying this because of the Invulnerability.

**Comment by MF366-Coding** [comment]
Thanks a lot, @norbcodes for your contributions to marking 3 levels as **DONE**!
You're always ready to help out and that's awesome. 😁

File: .github/doom_ss/TNT/MAP22/tnt22_4.png
Comment (norbcodes) [reply .github/doom_ss/TNT/MAP22/tnt22_4.png:1]: Oops

**Comment by norbcodes** [comment]
B)

File: .github/doom_ss/TNT/MAP22/tnt22_4.png
Comment (MF366-Coding) [reply .github/doom_ss/TNT/MAP22/tnt22_4.png:1]: nah np tho LOL

**Pull request merged**